src/airflow/Dockerfile

```diff
@@ -1,23 +1,21 @@
 FROM apache/airflow:2.7.1-python3.8
 
+# Install additional Python requirements.
+# --no-cache-dir is a good practice when installing packages using pip, because it helps to keep the image lightweight.
 COPY requirements.txt /requirements.txt
-RUN pip install --user --upgrade pip
-RUN pip install --no-cache-dir --user -r /requirements.txt # --no-cache-dir good practise when installing packages using pip. It helps to keep the image lightweight
-
+RUN pip install --no-cache-dir --user --quiet --upgrade pip
+RUN pip install --no-cache-dir --user --quiet --requirement /requirements.txt
 
+# Install the GCP CLI.
 # Source: https://airflow.apache.org/docs/docker-stack/recipes.html
-# Installing the GCP CLI in the container
 SHELL ["/bin/bash", "-o", "pipefail", "-e", "-u", "-x", "-c"]
-
 USER 0
 ARG CLOUD_SDK_VERSION=322.0.0
 ENV GCLOUD_HOME=/home/google-cloud-sdk
-
 ENV PATH="${GCLOUD_HOME}/bin/:${PATH}"
-
 RUN DOWNLOAD_URL="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz" \
     && TMP_DIR="$(mktemp -d)" \
-    && curl -fL "${DOWNLOAD_URL}" --output "${TMP_DIR}/google-cloud-sdk.tar.gz" \
+    && curl --silent -fL "${DOWNLOAD_URL}" --output "${TMP_DIR}/google-cloud-sdk.tar.gz" \
```

Comment:
Is this needed for command line GCP related commands? Is the Google Cloud Storage Provider not suitable for the task?

Reply:
The GCP installation part is actually left over from the existing implementation. I know that my DAG definitely doesn't need it, and from looking at the refactored existing DAG, it shouldn't need that either. I agree that the Airflow provider should be able to take care of everything. So I went ahead and removed it.